web-workout-tracker/workout/src/reducers/index.js

```diff
@@ -8,10 +8,12 @@ import {
     LOGIN_FAILURE,
     UPDATE_USER_START,
     UPDATE_USER_SUCCESS,
-    UPDATE_USER_FAILURE
+    UPDATE_USER_FAILURE,
+    SET_USER_STORE_SUCCESS
 } from '../actions';
 
 const initialState = {
+    thisUser: [],
```

Comment:
do we really need second user object in state? User is the one for storing user data